test/dtDpgNtuples_2018ZB_cfg.py

```diff
@@ -0,0 +1,152 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
It is OK to add ZB and Express cfgs, but step2 are probably too much.
For the cfgs you add please remove the commented parts when they are not useful.